libssu/ssufeaturemanager.cpp

```diff
@@ -57,8 +56,7 @@ QStringList SsuFeatureManager::repos(bool rndRepo, int filter)
 
 QString SsuFeatureManager::url(const QString &repo, bool rndRepo)
 {
-    QString repoHeader = QString("repositories-%1/")
-                         .arg(rndRepo ? "rnd" : "release");
+    QString repoHeader = QString("repositories-%1/").arg(rndRepo ? "rnd" : "release");
```

Comment:
I think in such instances QStringLiteral should be used so QString isn't allocated twice.

Reply:
No change here, I just merged the lines.

Besides that, it's not that simple. QStringLiteral will create an instance of the QString at compile time, optimizing for speed instead of binary size. There's also two instances of these strings where the compiler should be able to combine to use the same while with QStringLiteral it maybe cannot. QLatin1String maybe best here, but doesn't really make that much difference.

Reply:
OK, just forwarding the clazy warnings I have seen for such instances. The size increases in the instances I've used where very minimal e.g. one instances was 1k extra file size (unstripped binary).

I see now that you only reformatted the lines. Well anyway if you would have touched them it did seem simple enough of a improvement todo.